packages/components/react-web/src/Icon/icons/IconEdit.jsx

```diff
@@ -0,0 +1,39 @@
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+// DO NOT MODIFY THIS FILE; IT WAS GENERATED BY buildIconComponents.mjs.
+// =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import PropTypes from "prop-types";
+import iconStyles from "../Icon.styles";
+
+function IconCamera(props) {
+  return (
```

Comment:
This file needs to be generated
I don't see changes that would (re)generate this file

See how `buildIconComponents.mjs` 
it uses `const iconSources = elements.icons;` to generate React components
1. the `edit` icon needs to be added to `packages/elements/src/icons/svg`
2. then `packages/elements/src/icons/index` should be rebuilt (buildIndexJs.mjs)
3. then `react-web` Icons rebuilt - `buildIconComponents.mjs` 
4. Finally everything committed

Reply:
Sorry for any confusion and for the lack of documentation around this! I’ll add a readme in this folder that explains the process.